config/rbac/role.yaml

```diff
@@ -5,6 +5,14 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
This was removed when the RLP work was removed. Looks to have been added back since https://github.com/Kuadrant/multicluster-gateway-controller/commit/bf3c6e112a4172d696adceb49675bcfc1e6075f9#diff-6afa7d73841a5c0ebdb78eaf859ce5a400203a037f871440940b5347c3a97376R96. Seeing the same locally. We should probably have a test that runs the generate commands and checks nothing changed.

Reply:
Added issue for a CI job to catch these https://github.com/Kuadrant/multicluster-gateway-controller/issues/236